ui/src/translations/en.js

```diff
@@ -0,0 +1,2 @@
+module.exports = {
```

Comment:
Can you add a comment explaining why it's empty?

Also, it would be helpful if there's ui/src/translations/README.md which explains the format of files in the directory. It can be done later though.

Reply:
Also, I believe you need copyright & license header for every file.

Reply:
Added a comment about why it's empty.
About adding docs: noted. I'll do that when I write a script to get messages to TC.
About license/copyright: see thread on pull request #466.